pytorch_lightning/core/lightning.py

```diff
@@ -122,6 +116,14 @@ def __init__(self, *args, **kwargs):
     def example_input_array(self) -> Any:
         return self._example_input_array
 
+    @property
+    def current_epoch(self):
+        return self.trainer.current_epoch if self.trainer else 0
```

Comment:
- what happens if the module is loaded from checkpoint? 
- should these be -1 if `self.trainer` isn't set? that could indicate that this is currently outside of training

Reply:
This could be a nice addition. 
- have set it to `0` as previously it was `self.current_epoch = 0` in `__init__`